### PR TITLE
Register skibidi.is-a.dev

### PR DESCRIPTION
**Pull request by bananaking6**
Register skibidi.is-a.dev with CNAME record pointing to bananaking6.github.io.

### CODE REVIEW

**Review by triach-rold — COMMENTED** [review]
Seems complete ? It's reachable but I have no idea how to play the game.

**Review by MaskDuck — DISMISSED** [review]

**Comment by bananaking6** [comment]
I have added more contact information.

File: domains/skibidi.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "bananaking6",
+        "email": "null"
```
[the remark this answers is not in the document]
Comment (bananaking6) [reply domains/skibidi.json]: I did

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/skibidi.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "bananaking6",
+        "discord": "skibidibagel"
```
[the remark this answers is not in the document]
Comment (bananaking6) [reply domains/skibidi.json]: all done

**Comment by bananaking6** [comment]
The email line has been added

File: domains/skibidi.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "bananaking6",
+        "discord": "skibidibagel",
+        "email": "null"
```
Comment (CuteDog5695) [inline domains/skibidi.json:5]: take out the null part please

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/skibidi.json (hunk 3 above)
Comment (bananaking6) [reply domains/skibidi.json]: mk

**Review by iostpa — APPROVED** [review]
seems fine to me now

**Pull request merged**